File: app/src/main/java/com/aistra/hail/utils/AppMetadataDatabase.java
package com.aistra.hail.utils;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {AppMetadataEntity.class}, version = 2, exportSchema = false)
public abstract class AppMetadataDatabase extends RoomDatabase {
    public abstract AppMetadataDao appMetadataDao();
}

File: app/src/main/kotlin/com/aistra/hail/utils/AppMetadataDatabase.kt
package com.aistra.hail.utils

import androidx.room3.Database
import androidx.room3.RoomDatabase

@Database(
    entities = [AppMetadataEntity::class, ActionEntity::class, ActionDependencyEntity::class],
    version = 5,
    exportSchema = false
)
abstract class AppMetadataDatabase : RoomDatabase() {
    abstract fun appMetadataDao(): AppMetadataDao
    abstract fun actionDao(): ActionDao
}
